### PR TITLE
Further small updates to examples, quickmodel, and pymcmodels

### PR DESCRIPTION
**Pull request by sonyahanson**
In this PR I hope to tackle a few small updates:
 - update examples to not have `env PYTHONPATH="./"` 
 - fix parsing in `quickmodel` to be slightly more flexible, currently it seems to stop at 4 ligands, which is not how our 384-well plates are organized
 - Remove `AdaptiveMetropolis` in pymcmodels
 - Change dLstated and dPstated to be provided in input files in `quickmodel`

### CODE REVIEW

File: AssayTools/pymcmodels.py [hunk 1]
```diff
@@ -779,8 +779,8 @@ def run_mcmc(pymc_model, nthin=20, nburn=0, niter=20000, map=True, db='ram', dbn
         mcmc.use_step_method(pymc.Metropolis, getattr(pymc_model, 'epsilon_ex'), proposal_sd=10000.0, proposal_distribution='Normal')
     if hasattr(pymc_model, 'epsilon_em'):
         mcmc.use_step_method(pymc.Metropolis, getattr(pymc_model, 'epsilon_em'), proposal_sd=10000.0, proposal_distribution='Normal')
-    # log_F_PL is highly correlated with DeltaG
-    mcmc.use_step_method(pymc.AdaptiveMetropolis, [pymc_model.log_F_PL, pymc_model.DeltaG], scales={ pymc_model.log_F_PL : 0.1, pymc_model.DeltaG : 0.1  })
+    # Uncomment below to use log_F_PL highly correlated with DeltaG to improve sampling somewhat.
+    #mcmc.use_step_method(pymc.AdaptiveMetropolis, [pymc_model.log_F_PL, pymc_model.DeltaG], scales={ pymc_model.log_F_PL : 0.1, pymc_model.DeltaG : 0.1  })
```
Comment (jchodera) [inline AssayTools/pymcmodels.py:783]: I think we have to add back `Metropolis` methods for `log_F_PL` and `DeltaG`.

Comment (jchodera) [reply AssayTools/pymcmodels.py:783]: That would be something like
```python
mcmc.use_step_method(pymc.Metropolis, getattr(pymc_model, 'log_F_PL'), proposal_sd=0.1, proposal_distribution='Normal')
mcmc.use_step_method(pymc.Metropolis, getattr(pymc_model, 'DeltaG'), proposal_sd=0.1, proposal_distribution='Normal')
```

Comment (sonyahanson) [reply AssayTools/pymcmodels.py:783]: Aren't they still there? https://github.com/choderalab/assaytools/blob/master/AssayTools/pymcmodels.py#L771-L772

Comment (jchodera) [reply AssayTools/pymcmodels.py:783]: Oh! Right, thanks!

Comment (sonyahanson) [reply AssayTools/pymcmodels.py:783]: Yes, those exact lines were never removed and are still at lines 771-2.

**Comment by sonyahanson** [comment]
All these changes are done!  You can now define `P_error` in your inputs file, as is now in the `inputs_example.py` in `scripts`.

Mergeable when checks pass.

**Comment by jchodera** [comment]
Woohoo!

**Pull request merged**